packages/server/src/plugin/usageReporting/stats.ts

```diff
@@ -61,20 +61,30 @@ export class OurReport implements Required<IReport> {
     trace,
     asTrace,
     referencedFieldsByType,
+    // The max size a trace can be before it is sent as stats. Note that the
```

Comment:
I think I'd replace the second sentence with something like "Note that the trace sampling algorithm in the Apollo reporting ingress server will never select traces over 10mb anyway" or something. (It's not that the server entirely ignores them (as you say in the second half of the sentence), just that this logic here matches similar logic there.)

Reply:
I took a stab at the wording, feel free to change it again if you like.